spec/features/idv/phone_otp_rate_limiting_spec.rb

```diff
@@ -99,8 +99,6 @@ def expect_rate_limit_circumvention_to_be_disallowed(user)
   end
 
   def expect_rate_limit_to_expire(user)
-    Throttle.where(throttle_type: :idv_doc_auth).destroy_all
```

Comment:
I feel like for the redis world we could jsut delete everything in the throttle namespace?

Reply:
This interestingly didn't need the deletion to begin with since this file is primarily testing the `OtpRateLimiter`, not `Throttle`